public/mobile-app/src/routes/address/+page.svelte

```diff
@@ -120,9 +148,11 @@
                    autocomplete="address-line1"
                    oninput={addressInputHandler}
             >
-            {#if addressHasError}
+            {#if addressInputHasError}
               <div class="fr-messages-group" aria-live="polite">
-                <p id="address-error" class="fr-message fr-message--error" data-testid="address-error">Merci d'indiquer une adresse valide. L'adresse doit contenir entre 3 et 200 caractères et commencer par un nombre ou une lettre.</p>
+                <p id="address-error" class="fr-message fr-message--error" data-testid="address-error">
+                  Cette adresse est invalide. Conseil : saisissez entre 3 à 200 caractères et commencez par un nombre ou une lettre.
```

Comment:
Nit: the space before a `:` is a "espace insécable", so it's good practice to use one:


```suggestion
                  Cette adresse est invalide. Conseil&nbsp;: saisissez entre 3 à 200 caractères et commencez par un nombre ou une lettre.
```

Reply:
Thanks, I add it!